inc/admin/attachments/namespace.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * @author Brad Payne, Alex Paredes
+ * @license  GPLv3 (or any later version)
+ */
+
+namespace Pressbooks\Admin\Attachments;
+
+use Pressbooks\Licensing;
+use Pressbooks\Utility;
+
+/**
+ * Hooks into 'attachment_fields_to_edit' filter to add custom attachment
+ * metadata. Currently, on the display side of capturing this metadata
+ * unlike images, there is no easy way to determine an audio or video post id
+ * from the markup that's generated: (wp-image-33), therefore this can only
+ * apply to images, for now
+ *
+ * @since 5.5.0
+ *
+ * @param $form_fields
+ * @param $post
+ *
+ * @return mixed
+ */
+function add_metadata_attachment( $form_fields, $post ) {
+
+	if ( substr( $post->post_mime_type, 0, 5 ) === 'image' ) {
+
+		$author      = get_post_meta( $post->ID, 'pb_media_attribution_author', true );
```

Comment:
@alex-418 I think in your most recent changes the equals signs got aligned again; this isn't part of the Human Made coding standards we like so if you could revert this (so one space on either side of the `=` in variable assignments) that would be great. Thanks in advance!